### PR TITLE
Add spec "/etc/foreman-installer/scenarios.d/satellite.yaml"

### PR DESCRIPTION
**Pull request by huali027**
Signed-off-by: Huanhuan Li <huali@redhat.com>

### All Pull Requests:

Check all that apply:

* [x] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [ ] Is this PR an enhancement?

### CODE REVIEW

File: insights/parsers/satellite_yaml.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        :lock_package_versions: true
+    :facts:
+        tuning: medium
+        mongo_cache_size: 6.25
+
+Examples:
+    >>> ':facts' in SatelliteYaml
+    True
+    >>> SatelliteYaml[':facts']['tuning']
+    'default'
```
Comment (xiangce) [inline insights/parsers/satellite_yaml.py:22]: This result of the example is different from the above sample input. Could you please make them consistent?

File: insights/specs/default.py [hunk 2]
```diff
@@ -621,6 +621,7 @@ def pmlog_summary_file(broker):
     satellite_custom_ca_chain = simple_command(
         '/usr/bin/awk \'BEGIN { pipe="openssl x509 -noout -subject -enddate"} /^-+BEGIN CERT/,/^-+END CERT/ { print | pipe } /^-+END CERT/ { close(pipe); printf("\\n")}\' /etc/pki/katello/certs/katello-server-ca.crt',
     )
+    satellite_custom_hiera = simple_file("/etc/foreman-installer/custom-hiera.yaml")
```
Comment (xiangce) [inline insights/specs/default.py:624]: Is this line has something with this PR? Otherwise please modify it in another PR. Thanks.

**Review by xiangce — CHANGES_REQUESTED** [review]

File: insights/parsers/satellite_yaml.py (hunk 1 above)
Comment (huali027) [reply insights/parsers/satellite_yaml.py:22]: @xiangce, sure.

File: insights/specs/default.py (hunk 2 above)
Comment (huali027) [reply insights/specs/default.py:624]: @xiangce, no relationship, I just found it's disordered when I add this one, sure, I'll update it in another PR.

**Review by xiangce — APPROVED** [review]

**Pull request merged**